gradle/libs.versions.toml

```diff
@@ -89,7 +89,8 @@ timber = "5.0.1"
 coroutines = "1.4.2"
 
 # Local Server
-ktor = "1.6.0"
+ktor = "1.6.8"
+ktorServer = "3.0.0-rc-1"
```

Comment:
is there a concern if we are using a `rc` here ?

Reply:
This lib is only used for the sample app, so it's ok